client/app/core/session.service.js

```diff
@@ -30,24 +29,33 @@ export function SessionFactory ($http, $q, $sessionStorage, $window, $state, $co
     $sessionStorage.token = model.token
   }
 
-  function setMiqGroup (group) {
+  function setGroup (group) {
     $http.defaults.headers.common['Accept'] = 'application/json;charset=UTF-8'
-    $http.defaults.headers.common['X-Miq-Group'] = unescape(encodeURIComponent(group))
-    $sessionStorage.miqGroup = group || null
-    $sessionStorage.selectedMiqGroup = group
+    if (typeof group === 'object') {
```

Comment:
i know this is ugly,
> why not just do `foo = group.stuff || group` ???

I WANT TO but our tests that rely on this work fail when this is written shorthand, i think it has something to do with running in strict mode 🤔